packages/components/src/components/chat-search/Instructions.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="instructions">
+    <h2>Navie, the AppMap AI</h2>
+    <div>
+      <p>
+        Navie is an AI software architect. You ask Navie questions about your code, and she provides
+        comprehensive answers about code architecture and design. You can also work with Navie to
+        design new features, and to improve the performance and security of your code.
+      </p>
+    </div>
+    <div>
+      <h3>How Navie Works</h3>
+      <p>
+        Navie is able to think like a software architect because she has access to your AppMaps.
+        AppMaps are a new kind of software artifact that describe the runtime behavior of your
+        application.
+      </p>
+      <p>
+        When you ask Navie a question, she starts by searching your AppMaps to find the ones that
+        are most relevant to your question. Then the information in the AppMaps is used to find
+        relevant code snippets from across the entire code base. Finally, the AppMaps and code
+        snippets are examined by Navie to provide a comprehensive answer to your question.
+      </p>
+    </div>
+    <div v-if="appmapStats">
+      <h3>Your AppMaps</h3>
+      <p v-if="appmapStats.numAppMaps">
+        You have <strong>{{ appmapStats.numAppMaps }}</strong> AppMaps in your workspace. These
+        AppMaps contain <strong>{{ appmapStats.packages.length }}</strong> packages and
+        <strong>{{ appmapStats.classes.length }}</strong> classes.
+        <span v-if="appmapStats.routes.length"
+          >Your AppMaps contain <strong>{{ appmapStats.routes.length }}</strong> HTTP server routes
+        </span>
+        <span v-if="appmapStats.tables.length">
+          <span v-if="appmapStats.routes.length">and </span>
+          <span v-else>Your AppMaps contain </span
+          ><strong>{{ appmapStats.tables.length }}</strong> SQL tables</span
+        >.
+      </p>
+      <p data-cy="no-appmaps" v-else>
+        ⚠️ You don't have any AppMaps in your workspace. Before you can use Navie, you'll need to
+        create some.
+      </p>
+    </div>
+    <div class="tips">
+      <h3>Tips</h3>
+      <ul>
```

Comment:
The 'Tips' section conflicts with the suggestions in the left hand column. If we are unsure if people have enough AppMap data then we shouldn't be suggesting they look for Security issues, etc. These messages feel really discordant.

Reply:
Again, I am not trying reconcile the entire design, just responding to the issues that people raised when they were testing this UI. There are things to know about Navie that make it different from ChatGPT or Copilot. Somehow we need to get those across.